src/pilot/deploy-overcloud.py

```diff
@@ -718,9 +357,16 @@ def main():
         # Edit the dellnfv_environment.yaml
         # If disabled, default values will be set and
         # they won't be used for configuration
-        edit_dell_environment_file(args.enable_hugepages, args.enable_numa,
-                                   args.hugepages_size, vcpu_pin_set,
-                                   args.num_computes)
+        # Create ConfigOvercloud object
+        config = ConfigOvercloud(args.overcloud_name)
+        # Remove this when Numa siblings added
+        # Edit the dellnfv_environment.yaml
+        config.edit_dell_environment_file(
+            args.enable_hugepages,
+            args.enable_numa,
+            args.hugepages_size,
```

Comment:
hugepage_size should have a default value, otherwise this call will fail if no value has been passed to the parser above.

Reply:
Please refer my above comment

Reply:
What about a manual deployment?

Reply:
Fixed
Added default values.